### PR TITLE
*: Update manifests, scripts and docs to use 4.3

### LINKED ISSUE #952: olm data needs to be updated for 4.3
https://github.com/operator-framework/operator-metering/tree/release-4.3
file not found: manifests/deploy/openshift/olm/bundle/4.3/image-references

### PR DESCRIPTION
**Pull request by chancez**
Updates all manifests and images to reference 4.3

Closes #952 

/hold Waiting to make sure that we no longer need the 4.2 manifest bundles in the repo

### CODE REVIEW

**Comment by chancez** [comment]
Need to wait for https://github.com/openshift/release/pull/5205 to merge cause this uses 4.3 image tags in quay that aren't yet mirrored yet.

File: hack/get-metering-package-images.sh [hunk 1]
```diff
@@ -2,4 +2,6 @@
 
 # kubectl -n openshift-marketplace port-forward svc/art-applications 50051
 
-grpcurl -plaintext -d '{"pkgName": "metering-ocp", "channelName": "4.2"}' localhost:50051 api.Registry/GetBundleForChannel  | jq '.csvJson' -r | jq '.spec.install.spec.deployments[0].spec.template.spec.containers[] | select(.name=="operator") | .env | map(select(.name | match(".*_IMAGE"))) | map(.value |= (. | ltrimstr("image-registry.openshift-image-registry.svc:5000/"))) | .[] | "export " + .name + "=" + .value' -r
+CHANNEL="${CHANNEL:-4.3}"
+
+grpcurl -plaintext -d '{"pkgName": "metering-ocp", "channelName": "'"$CHANNEL"'"}' localhost:50051 api.Registry/GetBundleForChannel  | jq '.csvJson' -r | jq '.spec.install.spec.deployments[0].spec.template.spec.containers[] | select(.name=="operator") | .env | map(select(.name | match(".*_IMAGE"))) | map(.value |= (. | ltrimstr("image-registry.openshift-image-registry.svc:5000/"))) | .[] | "export " + .name + "=" + .value' -r
```
Comment (timflannagan1) [inline hack/get-metering-package-images.sh]: I guess we could also make the package name a variable too. Not sure how often that package name will be changed however.

Comment (chancez) [reply hack/get-metering-package-images.sh]: It shouldn't change, so I figured i would leave that out until we need that.

**Pull request merged**